lib/model.php

```diff
@@ -76,10 +80,18 @@ public function purge()
      */
     private function _getStore()
     {
+        // FIXME
+        // Workaround so that config value don't need to be changed
```

Comment:
I think @elrido should say how/whether we want to keep backward-compatibility here. We also have a config option for backward-compatibility, so probably we could use this although it may not be useful in this case.